cvat/apps/engine/models.py

```diff
@@ -453,7 +453,7 @@ def get_tmp_dirname(self) -> str:
 
 
 @transaction.atomic(savepoint=False)
-def clear_annotations_in_jobs(job_ids):
+def clear_annotations_in_jobs(job_ids: Iterable[int] | Iterator[int]):
```

Comment:
`Iterator` is a subclass of `Iterable`, so this is equivalent to `Iterable[int]`.